package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "fastify-jitson",
+  "version": "1.0.0-alpha.0",
+  "description": "A fastify plugin to use jitson as default parser",
+  "main": "plugin.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "tap test/*.js",
+    "lint": "standard --verbose | snazzy",
+    "bench": "branchcmp -r 2 -g -s \"npm run benchmark\"",
+    "benchmark": "npx concurrently -k -s first \"node ./example/jitson.js\" \"npm run benchmark:request\"",
+    "benchmark:core": "npx concurrently -k -s first \"node ./example/core.js\" \"npm run benchmark:request\"",
+    "benchmark:request": "npx autocannon -c 100 -d 5 -p 10 -m POST -b '{ \"hello\": \"world\" }' -H Content-Type=application/json localhost:3000/"
```

Comment:
You'll likely need to do two or three runs to get stable numbers. What's the difference between core and jitson?

Reply:
Screenshots of 4th run

```bash
npm run benchmark
```
![Screenshot from 2019-03-26 14-16-24](https://user-images.githubusercontent.com/15520377/55000136-24c30d80-4fd2-11e9-9825-0e6a7f1e831c.png)

```bash
npm run benchmark:core
```
![Screenshot from 2019-03-26 14-17-18](https://user-images.githubusercontent.com/15520377/55000167-373d4700-4fd2-11e9-8d60-beb2bdea75e5.png)